### DISCUSSION TITLE
Reset Filters Button

### DISCUSSION THREAD

**Discussion by stioboranu** [microsoft-search/pnp-modern-search#2500] [Ideas]
Our team is always frustrated that in order to reset the filters they need to refresh the page. Lets say that I am applying 6 different types of filters to narrow the search and I want to dispose all of them at the same time. We can do that only by refreshing the page.  

In order to dispose all the applied filters at the same time a button named "Clear all" should be added in the PnP - Search Filters component to dispose all the filters at the same time.

**Comment by BVSanjay** [comment]
Waiting for this to be implemented

**Comment by JasmineClapton** [comment]
I have a client also looking for this enhancement - would be a great feature to include.

**Comment by kasperbolarsen** [comment]
In the meantime you can add a regular Button web part add have it call the URL of the page. If you add a # at the end of  the URL I *think* it will refresh the page without doing a reload

**Reply by JasmineClapton** [reply]
Unfortunately, the '#' trick doesn't work and does a full page reload.
